Basic-Scripts/String Validators.py

```diff
@@ -0,0 +1,24 @@
+#This is the program to find out if the string  contains: alphanumeric characters, alphabetical characters, digits, lowercase and uppercase characters.
+if __name__ == '__main__':
+    s = input()
+    res = ["False", "False", "False", "False", "False"]
+    for i in s:
```

Comment:
Make use of meaningful variables instead of `i`.

Reply:
Updated String Validators.py
please review the code.